src/kernel-install/90-loaderentry.install

```diff
@@ -105,9 +105,10 @@ install -g root -o root -m 0644 "$KERNEL_IMAGE" "$ENTRY_DIR_ABS/linux" || {
 }
 
 shift "$INITRD_OPTIONS_SHIFT"
-# All files listed as arguments, and staged files called "initrd*" are installed as initrds.
+# All files listed as arguments, and staged files starting with "initrd" are installed as initrds.
 for initrd in "$@" "${KERNEL_INSTALL_STAGING_AREA}"/initrd*; do
     [ -f "$initrd" ] || {
+        [ "$initrd" = "${KERNEL_INSTALL_STAGING_AREA}/initrd*" ] && continue
```

Comment:
I suspect seems unnecessary/wrong; the `[ -f "$initrd" ]` check above will cover cases where the glob does not match any files.